libs/components/toast/src/lib/modules/toast/toast.component.ts

```diff
@@ -45,13 +46,18 @@ export class SkyToastComponent implements OnInit, OnDestroy {
    * Specifies a `SkyToastType` type for the toast to determine the color and icon to display.
    */
   @Input()
-  public set toastType(value: SkyToastType) {
-    this._toastType = value;
+  public set toastType(value: SkyToastType | undefined) {
+    if (value === undefined) {
+      this.toastTypeOrDefault = SKY_TOAST_TYPE_DEFAULT;
+    } else {
+      this.toastTypeOrDefault = value;
+    }
     this.updateIcon();
-  }
 
-  public get toastType(): SkyToastType {
-    return this._toastType === undefined ? SkyToastType.Info : this._toastType;
+    this.ariaLive =
```

Comment:
Wouldn't the ARIA live and ARIA role properties also need to be set when the initial state of the component is established with the default toast type? If so, I'd rename `updateIcon()` to `updateForToastType()` and move this logic there.

Reply:
In theory - the defaults that I set on the fields are all that are needed as they are correct for the default. However, I don't think what you suggested hurts and does give us some confidence so I went ahead and did that.